src/test/java/com/epam/library/data/factory/SpecificationFactoryImplTest.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.library.data.factory;
+
+import com.epam.library.data.BooksField;
+import com.epam.library.data.specification.Specification;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SpecificationFactoryImplTest {
+
+    @Test
+    public void testCreateShouldReturnEntity() {
+        SpecificationFactory factory = new SpecificationFactoryImpl();
+        Specification<String> specification = factory.create(BooksField.AUTHOR_NAME);
```

Comment:
test that Specification is an instance of AuthorSpecification, not just not null